app/views/users/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <div class='container'>
+      <% @users.each do |user| %>
+        <div class='card'>
+          <img class='userpic' src= <%= user.photo %> >
+          <a href="/users/<%= user.id %>" class='leftcard'>
+            <h2 class='username'><%= user.name %></h2>
+            <p class='count'>Number of posts: <%= user.posts_counter >= 0 ? user.posts_counter : '0' %></p><br>
+          </a>
+        </div>
+      <% end %>
+    </div>
+  </body>
```

Comment:
- [ ] I noticed that There is no comments view file (views/comments/new.html.erb) which accepts input for creating comments form. Please ensure to create a view with form in the views/comments/new.html.erb file as  required for this project and make the app works as expected :+1: